core/cap-0034.md

```diff
@@ -23,19 +23,22 @@ chosen `<close time, transaction set>` pair.
 
 With the current protocol, the following sequence of events can occur:
 
-- A transaction `T` with an expiration time makes it into a candidate
-`StellarValue`'s transaction set, with its expiration being no later than the
-`closeTime` in that `StellarValue`.
-- That transaction set is nominated, but with a later `closeTime`, as some
-other candidate(s) had a later `closeTime`, and the current protocol selects
-the maximum.
-- The ledger-closing process begins, and the ledger header is updated to reflect
-the newly-chosen `closeTime`.
-- The ledger-closing process reaches the point of applying transactions.
+1. A transaction `T` with an expiration time makes it into a candidate
+`StellarValue`'s transaction set, with its expiration time being greater than
+(or equal to) the last ledger close time.
+
+2. That transaction set is nominated, but with a `closeTime` greater than
```

Comment:
Nitpick: strange sentence "In other words, T has raced into the exact ledger in which turned out to have been going to expire". Either simplify if you think it adds clarity, or remove entirely.

Reply:
I've tried to make it less strange and more clear.